### ISSUE TITLE
Cannot read property 'bootstrapSwitch' of undefined

### ISSUE THREAD

**Issue by zacstewart** [Panman8201/ember-bootstrap-switch#4]
[This line](https://github.com/Panman8201/ember-bootstrap-switch/blob/master/addon/components/bootstrap-switch.js#L268) is returning `undefined`. The underlying `this.get('element')` is returning `null`. As a result, using the "checked" property breaks this.

I can't figure out of the error is an Ember bug or specific to this addon, but I thought I'd share it here in hopes that someone else has seen it. I'm not doing anything unusual in my usage, I don't think:

``` emblem
= bs-switch value=advertisingSite on-switch-change='toggleAdvertisingSite' checked=false
```

**Comment by Panman82** [comment]
Yeah, I've had troubles using this with Ember 1.13 and plan on re-writing the component to take advantage of the new rendering hooks with Glimmer. I'll need this for a project I'm working on so look for a new version in the next week or so.

**Comment by Panman82** [comment]
FYI, this was fixed with 1.13.0 that was just released.
